.gitignore

```diff
@@ -1,3 +1,4 @@
+src/ontology/imports/*
```

Comment:
be careful when you do that to remove all import declarations from phenio-edit.owl

Reply:
Is there a way to get ODK to do that?

Reply:
No you will have to simply delete all imports statement from phenio-edit.owl manually